.travis.yml

```diff
@@ -5,4 +5,4 @@ jdk:
   - oraclejdk9
   - oraclejdk10
 install: echo 'skipped'
-script: ./mvnw test-compile
+script: ./mvnw test
```

Comment:
Somehow, the script returns 0. But tests don't pass without an access token.
```
Tests run: 52, Failures: 29, Errors: 15, Skipped: 2
[ERROR] There are test failures.
```

Reply:
I didn't knew that, is that what is mentioned in this issue https://github.com/seratch/jslack/issues/36 ?

Actually it was also a convenient way for me to start build again because it failed downloading maven at first execution.

Reply:
Anyway, I cannot merge this change. Except for this one, I'd love to merge your improvements. Please revert only this change + squash commits into one commit.